projects/core/components/data-list/data-list.style.less

```diff
@@ -20,7 +20,8 @@ tui-data-list {
     }
 
     &[data-list-size='s'] {
-        --tui-data-list-margin: 0;
+        /* stylelint-disable-next-line */
+        --tui-data-list-margin: 0rem;
```

Comment:
is [this](https://raw.githubusercontent.com/Tinkoff/taiga-ui/screenshot-bot-storage/__bot-screenshots/Tinkoff-taiga-ui-3284/3757076144-2.png) expected behavior ?

<img width="1777" alt="Screenshot 2022-12-22 at 15 08 41" src="https://user-images.githubusercontent.com/35179038/209131515-e545cc4f-3beb-4549-b2ee-92acb4107f2f.png">

Reply:
<img width="1010" alt="Screenshot 2022-12-22 at 15 10 31" src="https://user-images.githubusercontent.com/35179038/209131778-1508a843-dd00-4a0c-97cf-0059fe305afa.png">

Reply:
Yes, this PR fixes lack of this spacing for `s` size. It was happening due to incorrect styles linting + CSS calc.